modules/pipeline/providers/queuemanager/queue/v_capacity.go

```diff
@@ -17,11 +17,11 @@ package queue
 import (
 	"fmt"
 
+	"github.com/erda-project/erda/modules/pipeline/providers/queuemanager/pkg/queue/priorityqueue"
+	"github.com/erda-project/erda/modules/pipeline/providers/queuemanager/types"
```

Comment:
bad import

Reply:
why?

Reply:
there is spacing between

"github.com/erda-project/erda/pkg/strutil" and "github.com/erda-project/erda/apistructs".

you can check the full file.